quixstreams/dataframe/dataframe.py

```diff
@@ -229,6 +251,91 @@ def func(d: dict, state: State):
         stream = self.stream.add_filter(func)
         return self._clone(stream=stream)
 
+    def _groupby_key(
+        self, key: Union[str, DataFrameFunc]
+    ) -> Callable[[object], object]:
+        """
+        PEP8 hates when you assign lambdas to variables...so this is here.
```

Comment:
It doesn't seem to describe what the function is doing.
Please change it or remove it